package.json

```diff
@@ -7,8 +7,9 @@
     "prepublish": "npx in-publish && npm run build || echo",
     "deploy": "npm run build && npm test && cd ./lib/flagsmith/ && npm publish && cd ../../lib/react-native-flagsmith && npm publish && cd ../../lib/flagsmith-es && npm publish",
     "deploy:beta": "npm run build && npm test && cd ./lib/flagsmith/ && npm publish --tag beta && cd ../../lib/react-native-flagsmith && npm publish --tag beta && cd ../../lib/flagsmith-es && npm publish --tag beta",
-    "build": "rollup -c && node ./move-react.js",
-    "test": "jest --env=jsdom"
+    "build": "npm run generatetypes && rollup -c && node ./move-react.js",
+    "test": "jest --env=jsdom",
+    "generatetypes": "npm exec quicktype https://raw.githubusercontent.com/Flagsmith/flagsmith/feat/evaluation-context-schema/sdk/evaluation-context.json -- -o evaluation-context.ts --src-lang schema --just-types --prefer-types --nice-property-names"
```

Comment:
Shouldn't need npm exec since it's a depedency, regardless, this errors for me - same error mentioned [here](https://github.com/glideapps/quicktype/issues/1629)

Reply:
Couldn't get it to work without `npm exec`. I'm running `npm install --include=dev`. Is there something I'm missing?